karaf-maven-plugin 4.4.9 doesn't work with Maven 3.9.12

When using karaf-maven-plugin 4.4.9 with Maven 3.9.12, the following error happens:

```
[ERROR] The plugin org.apache.karaf.tooling:karaf-maven-plugin:4.4.9 has unmet prerequisites: 
[ERROR] 	Required Java version 22 is not met by current version: 17.0.17
```

This error doesn't happen with karaf-maven-plugin 4.4.8.

It's probably related to https://github.com/apache/maven/commit/3dd64abedbd7b19a6a9aaed0e46ab31993a09c3a#diff-16f5eb789335df0361757ddc117468e4762e60a178cdf0a0edc5ea8c1c5bbf48

--- a/tooling/karaf-maven-plugin/pom.xml
+++ b/tooling/karaf-maven-plugin/pom.xml
@@ -356,8 +356,12 @@
                 </dependencies>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-plugin-plugin</artifactId>
                 <version>3.15.2</version>
+                <configuration>
+                    <requiredJavaVersion>[17,)</requiredJavaVersion>
+                </configuration>
                 <executions>
                     <execution>
                         <id>default-descriptor</id>

--- a/tooling/karaf-services-maven-plugin/pom.xml
+++ b/tooling/karaf-services-maven-plugin/pom.xml
@@ -144,7 +144,12 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-plugin-plugin</artifactId>
+                <version>3.15.2</version>
+                <configuration>
+                    <requiredJavaVersion>[17,)</requiredJavaVersion>
+                </configuration>
                 <executions>
                     <execution>
                         <id>default-descriptor</id>

